packages/@aws-cdk/aws-gamelift-alpha/README.md

```diff
@@ -63,25 +62,28 @@ Through a game session queue system to let FlexMatch forms matches and uses the
 declare const queue: gamelift.GameSessionQueue;
 declare const ruleSet: gamelift.MatchmakingRuleSet;
 
-new gamelift.QueuedMatchmakingConfiguration(this, 'QueuedMatchmakingConfiguration', {
-  matchmakingConfigurationName: 'test-queued-config-name',
-  gameSessionQueues: [queue],
-  ruleSet: ruleSet,
-});
+new gamelift.QueuedMatchmakingConfiguration(
+  this,
+  "QueuedMatchmakingConfiguration",
+  {
+    matchmakingConfigurationName: "test-queued-config-name",
+    gameSessionQueues: [queue],
+    ruleSet: ruleSet,
+  }
+);
```

Comment:
this is not a typo and i'm not sure i like the formatting change you've put forth.